mod/consensus-types/pkg/types/attestation_data_test.go

```diff
@@ -26,6 +26,8 @@ import (
 
 	"github.com/berachain/beacon-kit/mod/consensus-types/pkg/types"
 	"github.com/stretchr/testify/require"
+	"github.com/berachain/beacon-kit/mod/primitives/pkg/bytes"
```

Comment:
Linter is failing. Could you please fix it.

Reply:
Done 😄

Reply:
Great! Thanks you :)